packages/frontend/src/server/routers/PublicApiRouter/getScalingSummaryApiData.ts

```diff
@@ -1,11 +1,12 @@
 import { getScalingApiEntries } from '~/server/features/scaling/summary/getScalingApiEntries'
 import { getTvsChart } from '~/server/features/scaling/tvs/getTvsChartData'
+import { optionToRange } from '~/utils/range/range'
 
 export async function getScalingSummaryApiData() {
   const [entries, data] = await Promise.all([
     getScalingApiEntries(),
     getTvsChart({
-      range: { type: '30d' },
+      range: optionToRange('30d'),
```

Comment:
What is this absurd code below:
https://github.com/l2beat/l2beat/blob/282b5a55ce6c8c0db721b8850327619d98fccf22/packages/frontend/src/server/routers/PublicApiRouter/getScalingSummaryApiData.ts#L18-L26